openprescribing/media/js/src/chart_formatters.js

```diff
@@ -37,22 +37,39 @@ var formatters = {
     if (org === 'all') {
       str = 'all practices in NHS England';
     } else {
-      var is_practices = (org == 'practice') ? true : false;
-      if (orgIds.length > 0) {
-        str = this._getStringForIds(orgIds, is_practices);
-        if (_.any(_.map(orgIds, function(d) {
- return d.id.length > 3;
-}))) {
-          str += ' <br/>and other ' + org + 's';
-          str += (org === 'practice') ? ' in CCG' : '';
+      if (org === 'practice' && orgIds.length > 0) {
+        str = this._getStringForIds(orgIds, true);
+        if (_.any(_.map(orgIds, function(d) { return d.id.length > 3; }))) {
+          str += ' <br/>and other practices in CCG';
         }
       } else {
-        str = is_practices ? 'all practices' : 'all CCGs';
+        if (orgIds.length > 0) {
+          str = this._getStringForIds(orgIds, false);
+          str += ' <br/>and other ' + this.getFriendlyOrgType(org) + 's';
+        } else {
+          str = 'all ' + this.getFriendlyOrgType(org) + 's';
+        }
       }
     }
     return str;
   },
 
+  getFriendlyOrgType: function(orgType) {
+    switch(orgType) {
+      case 'practice':
+        return 'practice';
+      case 'ccg':
+      case 'CCG':
+        return 'CCG';
+      case 'stp':
+        return 'STP';
+      case 'regional_team':
+        return 'regional team';
```

Comment:
We call it "Regional Team" in the measure pages.

Reply:
I think there's a distinction between title and non-title case here which previously only showed up with "practice/Practice" as it doesn't apply to CCG. I think we want to say e..g "compare with other regional teams" rather then "compare with other Regional Teams" though we use "Regional Team" as an axis label